src/main/java/io/jenkins/plugins/ml/IPythonInterpreterManager.java

```diff
@@ -57,6 +57,7 @@
     IPythonInterpreterManager(IPythonUserConfig userConfig) {
         this.userConfig = userConfig;
         mockInterpreterGroup.put("session_" + sessionId, new ArrayList<Interpreter>());
+        LOGGER.setLevel(Level.OFF);
```

Comment:
I believe this is effectively changing the `Logger` level. But it is important to confirm that it is not changing the level for the whole JVM.

So if we have another plug-in, or some part of Jenkins, using Zeppelin/GRPC/etc, we must not disable the logger level for their code too.

Can you confirm that @Loghijiaha ? Might be easier to do this change in a separate PR to avoid blocking this one as it's pretty much ready to be merged otherwise IMO.

Reply:
Yes, that makes sense @kinow. Let's merge this and do it in another PR